src/main/java/uk/gov/companieshouse/model/dto/DirectorRequest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.*;
+
+public class DirectorRequest {
+
+    @NotBlank
+    @Size(min = 1, max = 250)
+    private String name;
+
+    @NotBlank
+    @Email
+    private String email;
+
+    @JsonProperty("on_behalf_name")
```

Comment:
for camelCase properties, serialise these as snake_case to meet CH API guidelines